active/dceu.md

```diff
@@ -10,7 +10,7 @@ Delegated responsibilities TBD.
 
 - Chair: TBD
 - Co-Chair: TBD
-- Board Liaison (must be an active Board member; may be the same as Chair/Co-Char): TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
```

Comment:
can you extract this cross-repo typo fix to a separate PR?

Reply:
Sure https://github.com/django/dsf-working-groups/pull/7